com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingMain.raytrace

```diff
@@ -79,7 +79,11 @@ void RayGen()
     {
         // Compute the ray direction from those coordinates (fast path for zero aperture)
         directionWS = -normalize(mul(jitteredPixelCoord, (float3x3)_PixelCoordToViewDirWS));
+    #if (SHADEROPTIONS_CAMERA_RELATIVE_RENDERING == 0)
         cameraPosWS = _WorldSpaceCameraPos;
```

Comment:
We should use:

float3 GetPrimaryCameraPosition()
{
#if (SHADEROPTIONS_CAMERA_RELATIVE_RENDERING != 0)
    return float3(0, 0, 0);
#else
    return _WorldSpaceCameraPos;
#endif
}